faq.mdx

```diff
@@ -66,23 +74,25 @@ OpenTofu will work with existing state files up to those created with Terraform
 
 <AccordionItem id="providers" summary="Does OpenTofu work with all the providers Terraform works with?">
 
-OpenTofu will not have its own providers. Terraform providers have not altered their licenses, and the potential for such a change is virtually zero. OpenTofu works with the current Terraform providers, but it uses a separate registry.
+The [Terraform Plugin SDK v2](https://github.com/hashicorp/terraform-plugin-sdk) and the newer [Terraform Plugin Framework](https://github.com/hashicorp/terraform-plugin-framework) are both available under the [Mozilla Public License 2.0](https://spdx.org/licenses/MPL-2.0.html) (MPL-2.0), an [open-source license](https://opensource.org/licenses?ls=MPL-2.0) ([definition](https://opensource.org/osd)). This means that we can continue leveraging the existing providers and modules in OpenTofu without having to fork them.
+
+Terraform providers have not altered their licenses, and the potential for such a change is virtually zero. OpenTofu works with the current Terraform providers, but it uses a [separate registry](https://search.opentofu.org). If those circumstances change, we will have to adapt. But we are confident that the OpenTofu and its community will be able to resolve any issues that arise.
```

Comment:
I would elaborate a bit more on why it is very unlikely for providers to change their licenses:

```suggestion
Terraform providers are separate projects built by the community and supported by different companies and authors. They have not altered their licenses, and the potential for such a change is virtually zero. OpenTofu works with the current Terraform providers, but it uses a [separate registry](https://search.opentofu.org). If those circumstances change, we will have to adapt. But we are confident that the OpenTofu and its community will be able to resolve any issues that arise.
```

Reply:
This text was previously there and I didn't change it, but the same question crossed my mind during editing — _why?_

I imagine that the original text was put together quickly, but expanding on the topic while still remaining FAQ-length could be a good idea.

Then again, maybe this is a topic for a deeper _technical FAQ_ that was mentioned in the Slack #dev-guides channel. A place where we can go deep on technical topics, beyond the scope of the VP-friendly FAQ on the homepage.

Let me play with this.